examples/camera-controller/webrtc_manager/WebRTCManager.cpp

```diff
@@ -162,8 +173,9 @@ CHIP_ERROR WebRTCManager::Connnect(Controller::DeviceCommissioner & commissioner
     });
 
     mPeerConnection->onLocalCandidate([this](rtc::Candidate candidate) {
-        std::string candidateStr = std::string(candidate);
-        mLocalCandidates.push_back(candidateStr);
+        auto candidateStr               = std::string(candidate);
+        ICECandidateStruct iceCandidate = { CharSpan::fromCharString(candidateStr.c_str()) };
```

Comment:
This part looks very fishy.

Reply:
The candidate string is pushed into a string vector member that is later used to construct the ICECandidateStruct DataList.